### PR TITLE
LPS-130483 Remove the Feature Flag

### PR DESCRIPTION
**Pull request by miltonmc**
**Ticket**: [LPS-130483](https://issues.liferay.com/browse/LPS-123334)

### Related issues
The following issues must be fixed prior to remove the feature flag:
- [x] [LPS-131181](https://issues.liferay.com/browse/LPS-131181) Double click on fields is not working #193
- [x] [LPS-131184](https://issues.liferay.com/browse/LPS-131184) The changes on languages are not being reflected on Form Builder #195
- [x] [LPS-131185](https://issues.liferay.com/browse/LPS-131185) A message regarding detectClickOutside is being displayed on devTools console #195
- [x] [LPS-131200](https://issues.liferay.com/browse/LPS-131200) User cannot save a form on Web Content #195 
- [x] [LPS-131217](https://issues.liferay.com/browse/LPS-131217) An error message is shown for each field after creating a Fieldset with nested fields #190
- [x] [LPS-131271](https://issues.liferay.com/browse/LPS-131271) It's not possible to edit a field with options in the fieldset modal #196

### CODE REVIEW

**Comment by interaminense** [comment]
Just started reviewing :)

File: modules/apps/data-engine/data-engine-taglib/src/main/java/com/liferay/data/engine/taglib/servlet/taglib/DataLayoutBuilderTag.java [hunk 1]
```diff
@@ -177,15 +173,9 @@ private ServletContext _getModuleServletContext() {
...
-		if (DataLayoutTaglibUtil.isUsingNewFormProvider()) {
-			return DataLayoutTaglibUtil.resolveModule(
-				"data-engine-taglib/data_layout_builder/new-js/plugins/" +
-					value + "/index");
-		}
-
 		return DataLayoutTaglibUtil.resolveModule(
-			"data-engine-taglib/data_layout_builder/js/plugins/" + value +
-				"/index.es");
+			"data-engine-taglib/data_layout_builder/new-js/plugins/" + value +
```
Comment (diegonvs) [inline modules/apps/data-engine/data-engine-taglib/src/main/java/com/liferay/data/engine/taglib/servlet/taglib/DataLayoutBuilderTag.java:177]: Do you guys have a plan to move the content from the `new-js/` folder to `js/`?

Comment (miltonmc) [reply modules/apps/data-engine/data-engine-taglib/src/main/java/com/liferay/data/engine/taglib/servlet/taglib/DataLayoutBuilderTag.java:177]: After remove the feature flag we should do a cleanup on the code to remove the old stuff and merge both into the js.

**Comment by miltonmc** [comment]
ci:test:relevant

**Comment by interaminense** [comment]
Just started reviewing :)

**Comment by miltonmc** [comment]
ci:test:sf

**Comment by miltonmc** [comment]
ci:test:relevant

**Comment by miltonmc** [comment]
Closed in favor of #205

**Pull request closed without merging**